modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/InstrumentControllerSim.scala

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package edu.gemini.seqexec.server
+
+import java.util.concurrent.atomic.AtomicBoolean
+
+import edu.gemini.seqexec.model.dhs.ImageFileId
+import edu.gemini.seqexec.server.SeqexecFailure.SeqexecException
+import gov.aps.jca.TimeoutException
+import org.log4s.getLogger
+import squants.Time
+
+import scala.annotation.tailrec
+import scalaz.{EitherT, Show}
+import scalaz.concurrent.Task
+import scalaz.syntax.show._
+import scalaz.syntax.std.boolean._
+
+class InstrumentControllerSim(name: String) {
```

Comment:
One issue I often have with the simulators is that times are sometimes too short. This is sometimes a problem while doing development where you want e.g. observe to take longer to see the effects.

I normally just change the code locally, but perhaps there should be configurable waits on each of the actions so i'd only need to modify those values without needing to add the delays